src/components/organisms/footer/footer.scss

```
@@ -0,0 +1,84 @@
+@use "../../variables" as *;
+
+.footer {
+  text-align: center;
+  align-items: center;
+  color: $black;
+  font-size: $fontsize-s;
+
+  .contact {
+    border-bottom: 1px solid $gray_nobel;
+    padding: 52px 20px;
+    @media screen and(max-width:$breakpoint-md) {
+      padding: 32px 20px;
+    }
+    .wrapper {
+      padding-bottom: 20px;
+      img {
+        width: 150px;
+      }
+      adress {
+        line-height: 28px;
+      }
+    }
+    .icon-list {
+      display: flex;
+      justify-content: center;
+      gap: 14px;
+      .icon {
+        color: $black;
+
+        i {
+          width: 14px;
+          height: 14px;
+          transition: color 0.2s ease;
+          &:hover {
+            color: $gray_nobel;
+          }
+        }
+      }
+    }
+  }
+  nav {
+    padding: 16px;
+    background-color: $white_smoke;
+    ul {
+      margin-bottom: 0px;
+      padding: 0px;
+      list-style: none;
+      columns: 8;
+      column-rule: 1px solid;
+      column-gap: 8px;
+      @media screen and(max-width: $breakpoint-md) {
+        columns: 2;
+        column-rule: none;
+      }
+
+      li {
+        @media screen and(max-width: $breakpoint-md) {
+          text-align: start;
+          padding-bottom: 8px;
+        }
+      }
```

```suggestion
      li {
      padding: 0 8px;
      &:not(:last-child) {
        border-right: 1px solid #black;
      }
        @media screen and(max-width: $breakpoint-md) {
          text-align: start;
          padding-bottom: 0 0 8px 0;
          border: 0;
        }
      }
```
区切り線の指定はこちらに移動させましょう。